app/styles/app.scss

```diff
@@ -1 +1,89 @@
+/* ==================================
+   #APPUNIVERSUM POC
+   ================================== */
+
+/**
+ * A design universe for Flanders GOV applications
+ */
+
+/**
+ * CONTENTS
+ * ---
+ * SETTINGS: Project settings and variables
+ * TOOLS: Mixins
+ * GENERIC: Reset, global box-sizing and font-styles
+ * ELEMENTS: Plain html tag styling
+ * OBJECTS: OOCSS unstyled objects (namespace: '.au-o-...')
+ * COMPONENTS: Components (namespace: '.au-c-...')
+ * UTILITIES: Single function helper classes (namespace: '.au-u-...')
+ * SHAME: Temporary hacks and quickfixes
+ */
+
+
+// VARIABLES
+@import "appuniversum/s-colors";
+@import "appuniversum/s-global";
+@import "appuniversum/s-utilities";
+
+
+// MIXINS
+@import "appuniversum/t-font-size";
+@import "appuniversum/t-sass-mq";
+
+
+// GENERIC
+@import "appuniversum/g-reset";
+@import "appuniversum/g-box-sizing";
+// @import "appuniversum/g-font"; // (BREAKING CHANGE | CONFLICT IN FONT WEIGHTS)
```

Comment:
I fixed this in AU and it's working fine in GN. Maybe check if this is also the case in this project.

Reply:
It also requires testing this in loket, because it will be imported there, and that is more extensive. Can we fix that when it is tested in loket?